x-pack/plugins/observability_solution/infra/public/pages/metrics/inventory_view/lib/create_inventory_metric_formatter.ts

```diff
@@ -65,6 +65,8 @@ const METRIC_FORMATTERS: MetricFormatters = {
   },
   rx: { formatter: InfraFormatterType.bits, template: '{{value}}/s' },
   tx: { formatter: InfraFormatterType.bits, template: '{{value}}/s' },
+  rxV2: { formatter: InfraFormatterType.bits, template: '{{value}}' },
+  txV2: { formatter: InfraFormatterType.bits, template: '{{value}}' },
```

Comment:
What happens if you add the `/s` here?
```suggestion
  rxV2: { formatter: InfraFormatterType.bits, template: '{{value}}/s' },
  txV2: { formatter: InfraFormatterType.bits, template: '{{value}}/s' },
```